doc/src/learn/about-sui.md

```diff
@@ -6,31 +6,33 @@ Sui is the first permissionless Layer 1 blockchain designed from the ground up t
 
 ## What Sui is
 
-Sui is a smart contract platform maintained by a permissionless set of validators that play a role similar to validators or miners in other blockchain systems.
+Sui is a [smart contract](sui-glossary#smart-contract) platform maintained by a permissionless set of [validators](sui-glossary#validator) that play a role similar to validators or miners in other blockchain systems.
```

Comment:
I don't think these links will work. You need to Markdown file extension, for sure.

Reply:
Thanks, I will confirm. I see this link in the same format working on the site:
https://github.com/MystenLabs/sui/blob/main/doc/src/learn/sui-glossary.md?plain=1#L121

But I also see a link to the same using .md:
https://github.com/MystenLabs/sui/blob/main/doc/src/learn/sui-glossary.md?plain=1#L19

Both work in public docs

Reply:
> Thanks, I will confirm. I see this link in the same format working on the site: https://github.com/MystenLabs/sui/blob/main/doc/src/learn/sui-glossary.md?plain=1#L121
> 
> But I also see a link to the same using .md: https://github.com/MystenLabs/sui/blob/main/doc/src/learn/sui-glossary.md?plain=1#L19
> 
> Both work in public docs

Yes, these will work on docs.sui.io but not in the GitHub Markdown version. I will fix the FAQ. Thanks for the catch!

Reply:
ah, thanks for the clarification. I can add the glossary file to this PR